README.md

```diff
@@ -2,10 +2,9 @@
 
 ###### Supports iOS 9.0 and up.
 
-[![Version](https://img.shields.io/github/release/iNDS-Team/iNDS.svg)](https://github.com/iNDS-Team/iNDS/releases/latest)
```

Comment:
If we could get this badge back, that would be great. Otherwise, everything looks good.

Reply:
Done.